### DISCUSSION TITLE
Add custom masks

### DISCUSSION THREAD

**Discussion by adrielschmitz** [typesnippet/antd-phone-input#105] [General]
The Brazilian standard is now 5 numbers and then 4... today the input is the opposite (4 before the space and 5 at the end). I think we could add a custom_mask logic and then the mask that should be used according to the country is passed, more or less like this:
```masks={{ br: '(..) .....-....' }}```

<img width="282" alt="Screenshot 2024-09-02 at 15 46 51" src="https://github.com/user-attachments/assets/5e7f9fb7-0fee-49d6-89fc-27b93a19a3ed">

**Comment by ArtyomVancyan** [comment]
Hi @adrielschmitz, please open a pull-request on [typesnippet/react-phone-hooks](https://github.com/typesnippet/react-phone-hooks) suggesting mask change for Brazil. Also, please put a wikipedia or other link that proves the correctness of the suggestion. I will review it.
